README.md

```diff
@@ -70,16 +70,44 @@ It's best if your certificates are signed by a Certificate Authority (CA) becaus
 This is not required though.
 You can generate and use a self-signed certificate.
 
-Here's an example command that generates a 2048-bit RSA certificate valid for a year:
+Here's an example command that generates a 2048-bit RSA certificate valid for a year.
+Before execute command, set shell variables: `$country`, `$state`, `$organization`, `$organization_unit`, `$domain_name`, `$email_address`
```

Comment:
Please reword:

```
Replace `$country`, `$state`, `$organization`, `$organization_unit`, `$domain_name`, `$email_address` with actual values.
```

Also, put it below the config file template, not here.

Reply:
Done.